src/components/styled/styles.tsx

```diff
@@ -149,3 +153,55 @@ export const FavBox = styled(Box)(({ theme }) => ({
   justifyContent: "center",
   alignItems: "center",
 }));
+
+export const StyledModal = styled(Modal)(() => ({
+  width: 520,
+  height: 550,
+  borderRadius: 20,
+  display: "flex",
+  justifyContent: "center",
+  flexDirection: "column",
+  alignItems: "center",
+  position: "fixed",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  border: "5px solid #ccc",
+  backdropFilter: "blur(10px)",
+}));
+
+export const StyledAccordion = styled(Accordion)(() => ({
+  backgroundColor: "transparent",
+  backgroundImage: "none",
+  width: 400,
+  height: 100,
+  display: "flex",
+  flexDirection: "column",
+  textAlign: "center",
+  textTransform: "uppercase",
+  margin: 50,
+  borderBottom: 0,
+  boxShadow: "none",
+  "&:last-child": {
+    borderBottom: 0,
+  },
+  "&:before": {
+    display: "none",
+  },
+}));
```

Comment:
If you are using only StyledModal and not default Modal component, you override this styles right in the Theme

https://youtu.be/k8nxTf7rYCE?si=4wJ5m6jRKTESYCRO